extensions/index.md

```diff
@@ -2,22 +2,101 @@
+Anyone can author a profile, and a single profile can be reused by multiple APIs.
+Popular profiles may be implemented by off-the-shelf tools so that developers 
+can seamlessly take advantage of the features these profiles provide.
+
+## <a href="#existing-profiles" id="existing-profiles" class="headerlink"></a> Existing Profiles
+
+{% for category in site.profile_categories %}
+  <h3 id="#profiles-category-{{ category | slugify }}">
+    {{ category }}
+  </h3>
+  <dl class="profiles-list">
+    {% for profile in site.profiles %}
+      {% if profile.categories contains category %}
+        <dt><a href="{% include profile_url.md page=profile %}">{{ profile.name }}</a></dt>
+        <dd>{{ profile.short_description }}</dd>
+      {% endif %}
+    {% endfor %}
+  </dl>
+{% endfor %}
+
+## <a href="#profile-creation" id="profile-creation" class="headerlink"></a> Creating a New Profile
+
+### <a href="#profile-creation-before" id="profile-creation-before" class="headerlink"></a> Before Creating a Profile
+
+Please **check whether an existing profile fits your needs or could be amended
+to fit your needs** before developing a new one.
+
+- If a suitable profile already exists, consider using it. Having fewer, more
+widely-deployed profiles makes it easier to create shared tooling.
+
+- If there's an existing profile that could be amented to fit your needs,
+consider asking the profile's author if they would be willing to modify it as
+needed. Contact them through the information in their profile's registration
+and give them some time to reply.
+
+### <a href="#profile-creation" id="profile-creation" class="headerlink"></a> Authoring Your Profile
+
+To author your profile, [download and fill out the template](/profile_template.md).
+
+### <a href="#profile-register" id="profile-register" class="headerlink"></a> Register & Use Your Profile
+
+Once it's ready, submit your profile to the JSON:API profile registry by creating
+a pull request that contains your filled out template as an `index.md` file in a 
+uniquely-named folder in the [`_profiles`](https://github.com/json-api/json-api/tree/gh-pages/_profiles) 
+directory.
+   
+Your submission will be **approved within a week** by one of JSON API's 
+editors, assuming it meets the [profile extension requirements](/format/1.1/#profiles-authoring)
+and follows the template above. The community may give you feedback on your
+submission before it's approved, but you are not required to act on this 
+feedback.
+
+One caveat: if your profile defines a new, fundamental mechanism for doing
+something "architectural" that other profiles may need to do too, it may be
+held for extra consideration. This extra review is designed to check that the
+proposed mechanism, were it to become a convention, wouldn't have problematic
+ramifications.
+
+Also, note that if the author of a registered profile dies, moves out of contact,
+or otherwise can't or won't make changes that are important to the community,
```

Comment:
I would simply say "becomes non-responsive for an extended period of time"

Reply:
I agree that this language could be a bit softer, if that's what you're going for.

For reference, I based it off the language used for media type registrations:

> The IESG may reassign responsibility for a media type. The most common case of this will be to enable changes to be made to types where the author of the registration has died, moved out of contact or is otherwise unable to make changes that are important to the community.

Maybe I should just copy that language directly?

Reply:
I just don't think it's necessary to give specific examples at all. "Legally" it's better to define a class of scenarios which trigger an action than it is to define specific scenarios and risk missing something thereby forcing yourself to interpret "the spirit of the law".

Reply:
See updated text.